geth-with-protocol/genesis.json

```diff
@@ -1,13 +1,13 @@
 {
   "config": {
     "chainId": 54321,
-    "homesteadBlock": 1,
-    "eip150Block": 2,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
     "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "eip155Block": 3,
-    "eip158Block": 3,
-    "byzantiumBlock": 4,
-    "constantinopleBlock": 5,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
```

Comment:
Could be a good idea to set the `istanbulBlock` to 0 as well.

Reply:
fixed in 6b564ea

I'll reword the commit in the rebase